src/breaking-change.ts

```diff
@@ -159,7 +180,10 @@ export async function runScript() {
 
     const resolved = resolvedMapForNewSpecs[swagger]
     if (resolved) {
-      const diffs = await runOad(path.resolve(pr!.workingDir,swagger), resolved);
+      const diffs = await runOad(
+        path.resolve(pr!.workingDir, swagger),
+        swagger // Since the swagger resolving  will be done at the oad , here to ensure the position output is consistent with the origin swagger,do not use the resolved swagger
```

Comment:
Is it related to oad report common parameters error?

Reply:
no，  it's just a improvement to the pipeline so that the location of the output message may be more clear . Currently the output  location indicate the resolved swagger location